report_compassion/models/res_partner.py

```diff
@@ -40,9 +40,16 @@ def get_receipt(self, year):
                 ("last_payment", "<=", end_date),
                 ("state", "=", "paid"),
                 ("product_id.requires_thankyou", "=", True),
+                # invoice from either the partner, the company or the employee
+                # to obtain the same results when tax receipt is computed from companies or employees
                 "|",
+                # invoice from the partner (when self is either an company or an employee)
                 ("partner_id", "=", self.id),
+                "|",
+                # invoice from the company (when self is an employee)
                 ("partner_id.parent_id", "=", self.id),
```

Comment:
@davidwul Should we remove this, in order for tax receipt to show 0.- if the invoice is at the company level?